src/modules/dashboard/components/ActionsPage/StakingWidget/StakingSlider.tsx

```diff
@@ -119,14 +135,18 @@ const StakingSlider = ({
         <Slider
           name="amount"
           value={values.amount}
-          limit={parseFloat(userStakeLimitPercentage.toFixed(2))}
+          limit={parseFloat(userStakeLimitPercentage.toFixed(5))}
```

Comment:
Why do you pass here 5 decimals after the point, when all across the slider we use 2 ?

Just look at the `step` value, that's what dictates the number of decimal points

Reply:
Otherwise it will be too imprecise - the limit value will be less or more than the activated tokens. This is just for the more precise limit value and it actually in my opinion supplies proper functionality.
In this example below (changed toFixed(2)), I have 50 activated tokens, but the limit is calculated as 50.1 and it goes passed the number of my activated tokens on the slider.
<img width="444" alt="Screenshot 2021-05-24 at 11 31 02" src="https://user-images.githubusercontent.com/34057551/119324662-fe639000-bc77-11eb-8006-e146f39d3f57.png">

There might be other instances where it will be less and then the user won't be able to stake all of his activated tokens.

Reply:
Sure